packages/date-picker/docs/FilterDateRangePicker.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react"
+import { ComponentStory, Story } from "@storybook/react"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { FilterDateRangePicker } from "../src/FilterDateRangePicker"
+import { DateRange } from "../src/types"
+import { defaultMonthControls } from "./controls/defaultMonthControls"
+import { disabledDayMatchersControls } from "./controls/disabledDayMatchersControls"
+import { dateRangePickerLocaleControls } from "./controls/localeControls"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.datePicker}/Filter Date Range Picker`,
+  component: FilterDateRangePicker,
+  parameters: {
+    actions: {
+      argTypesRegex: "^on.*",
+    },
+    docs: {
+      description: {
+        component:
+          'import { FilterDateRangePicker } from "@kaizen/date-picker"',
+      },
+    },
+  },
+  args: {
+    label: "Dates",
+    locale: "en-AU",
+  },
+  argTypes: {
+    ...dateRangePickerLocaleControls,
+    ...defaultMonthControls,
+    ...disabledDayMatchersControls,
+    description: {
+      control: "text",
+    },
+  },
+}
+
+export const DefaultStory: ComponentStory<
+  typeof FilterDateRangePicker
+> = props => {
+  const [range, setRange] = useState<DateRange | undefined>()
+
+  return (
+    <FilterDateRangePicker
+      {...props}
+      selectedRange={range}
+      onRangeChange={setRange}
+    />
+  )
+}
+DefaultStory.storyName = "Filter Date Range Picker"
+DefaultStory.parameters = {
+  docs: { source: { type: "code" } },
+}
+DefaultStory.args = {
+  id: "filter-drp--default",
+}
+
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => {
+  const [rangeDefault, setRangeDefault] = useState<DateRange | undefined>()
+  const [rangeExisting, setRangeExisting] = useState<DateRange | undefined>({
+    from: new Date("2022-05-15"),
+    to: new Date("2022-06-22"),
+  })
+
+  return (
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.Row rowTitle="Default">
+        <div>
```

Comment:
Our `StoryWrapper` component forces the contents into full width even if they aren't meant to. Will remove these `div`s when it is fixed ([draft in progress](https://github.com/cultureamp/kaizen-design-system/pull/3117)).